docs/deep-dive/03-dd-expose-function-serverless.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Expose a Function with an API Rule
```

Comment:
APIRule or API Rule? You used APIRule in the Overview, that's why I ask.

Reply:
Ok, let's go with APIRule (as discussed)